src/screens/WalletConnect/CallRequest/TransactionRequestContent.js

```diff
@@ -105,6 +105,7 @@ function TransactionRequestContent({ request, onConfirm, onReject }: Props) {
   const chainConfig = chainConfigs[chain];
   const value = wrapBigNumberOrNil(transactionPayload?.amount);
   const symbol = transactionPayload?.symbol;
+  const assetAddress = transactionPayload?.contractAddress ?? nativeAssetPerChain[chain].address;
```

Comment:
IMO we should not default to native asset, the same way we do not default to native asset symbol.

Reply:
Done